integration-tests/http/__tests__/claims/claims.spec.ts

```diff
@@ -595,23 +595,50 @@ medusaIntegrationTestRunner({
             adminHeaders
           )
 
+          // shipping Options w/ custom price
           const {
             data: {
               order_preview: { shipping_methods: outboundShippingMethods },
             },
           } = await api.post(
             `/admin/claims/${claimId}/outbound/shipping-method`,
-            { shipping_option_id: outboundShippingOption.id },
+            {
+              shipping_option_id: outboundShippingOption.id,
+              custom_price: 12.5,
+            },
             adminHeaders
           )
 
           const outboundShippingMethod = outboundShippingMethods.find(
             (m) => m.shipping_option_id == outboundShippingOption.id
           )
 
+          expect(outboundShippingMethod.subtotal).toBe(12.5)
+          expect(outboundShippingMethod.is_custom_amount).toBe(true)
+
+          // Reset shipping custom price
+          const {
+            data: {
+              order_preview: { shipping_methods: outboundShippingMethods2 },
+            },
+          } = await api.post(
+            `/admin/claims/${claimId}/outbound/shipping-method/${outboundShippingMethod.actions[0].id}`,
+            {
+              custom_price: null,
```

Comment:
q: should we rename this to: `cutom_amount` for consistency?

Reply:
yes, good point.. just renamed it